### DISCUSSION TITLE
Having trouble getting homebrew packages to build via JReleaser for a rust project

### DISCUSSION THREAD

**Discussion by manojkarthick** [jreleaser/jreleaser#715] [Q&A] [answered]
I updated my jreleaser-poc project to build homebrew packages using the Github Action. JReleaser does not throw an error but doesn't seem to build the packages. I tried using both global and local dist config, but that doesn't seem to affect the result.

Samples:

* Global config: [JReleaser config](https://github.com/manojkarthick/jreleaser-poc/blob/master/jreleaser.yml), [Github Actions Run](https://github.com/manojkarthick/jreleaser-poc/actions/runs/1752720403)
* Local distribution config: [JReleaser config](https://github.com/manojkarthick/jreleaser-poc/blob/9b35edeb97a520bf27988d94fab637f3779121dc/jreleaser.yml), [Github Action Run](https://github.com/manojkarthick/jreleaser-poc/actions/runs/1752693590)

Any idea what I'm missing to trigger the build?

**Comment by aalmiray** [comment, marked as answer]
Try using the `full-release` command instead of `release`. 
`release` only creates a Git release and nothing more.
`full-release` creates a Git release, publishes packagers, and announces the project.

Refer to the Workflow for more information. https://jreleaser.org/guide/latest/workflow.html

**Reply by manojkarthick** [reply]
Thank you! Got it to build and release the formula now, but the formula generated by default seems to generate a symlink to itself. Seems like it might be a bug.

Generated formula: https://github.com/manojkarthick/homebrew-tap/blob/master/Formula/jreleaser-poc.rb

```
❯ cd /opt/homebrew/Cellar/jreleaser-poc/0.2.5/libexec/bin 

❯ ls -alth
lrwxr-xr-x 13 B Wed Jan 26 20:55:27 2022 jreleaser-poc ⇒ jreleaser-poc
drwxr-xr-x 96 B Wed Jan 26 20:55:27 2022 ..
drwxr-xr-x 96 B Wed Jan 26 20:55:27 2022 .
```

**Reply by aalmiray** [reply]
Ah yes, this is a recent change I made and probably got it wrong 😓

**Reply by aalmiray** [reply]
Filed and fixed #716

**Reply by manojkarthick** [reply]
It works now, thank you so much!

**Reply by aalmiray** [reply]
@manojkarthick I'd like to write a follow up post on JReleaser + Rust with cross compilation enabled. May I base it on the work you'v done in `jreleaser-poc (with proper attribution of course)?

**Reply by manojkarthick** [reply]
Of course! That would be really useful! Thank you!

**Reply by aalmiray** [reply]
Awesome. Thanks a lot! 😄
